### PR TITLE
P11 attributes

### PR DESCRIPTION
**Pull request by microshine**

### CODE REVIEW

File: packages/cards/lib/card.json [hunk 1]
```diff
@@ -275,6 +275,11 @@
       "name": "Athena IDProtect - Cryptographic Java Card",
       "driver": "29a2bd60c4b841c686be68d2874c0c02"
     },
+    {
+      "atr": "3BDF18008131FE7D006B020C0182010101434E53103180EC",
```
Comment (microshine) [inline packages/cards/lib/card.json]: @defacto64 Please check it. Is it correct?

File: packages/cards/lib/card.json [hunk 2]
```diff
@@ -459,6 +469,14 @@
           "x64": "%WINDIR/SysWOW64/bit4ipki.dll"
         }
       }
+    },
+    {
+      "id": "DBADF7126B4E3656FA5E8CF2AE8D712DE953BC98",
```
Comment (microshine) [inline packages/cards/lib/card.json]: @defacto64 Please check it. Is it correct?

File: packages/cards/lib/card.json (hunk 1 above)
Comment (defacto64) [reply packages/cards/lib/card.json]: I have reviewed https://github.com/PeculiarVentures/webcrypto-local/blob/p11-attributes/packages/cards/lib/card.json and it seems correct to me, as regards reception of my issues #259 and #260.

File: packages/cards/lib/card.json (hunk 2 above)
Comment (defacto64) [reply packages/cards/lib/card.json]: I have reviewed https://github.com/PeculiarVentures/webcrypto-local/blob/p11-attributes/packages/cards/lib/card.json and it seems correct to me, as regards reception of my issues #259 and #260.

**Pull request merged**